model-client/src/commonMain/kotlin/org/modelix/model/client2/ReplicatedModel.kt

```diff
@@ -73,79 +66,92 @@ class ReplicatedModel(val client: IModelClientV2, val branchRef: BranchReference
             }
         }
 
-        rawBranch.addListener(object : IBranchListener {
+        // convergence watchdog
+//        scope.launch {
+//            var nextDelayMs: Long = 1000
+//            while (state != State.Disposed) {
+//                if (nextDelayMs > 0) delay(nextDelayMs)
+//                try {
+//                    val newRemoteVersion = remoteVersion.pull()
+//                    remoteVersionReceived(newRemoteVersion)
+//                    nextDelayMs = 1000
+//                } catch (ex: CancellationException) {
+//                    break
+//                } catch (ex: Throwable) {
+//                    LOG.error(ex) { "Failed to pull branch $branchRef" }
+//                    nextDelayMs = (nextDelayMs * 3 / 2).coerceIn(1000, 30000)
+//                }
+//            }
+//        }
```

Comment:
Can we put this into a separate method to avoid polluting the production code with commented-out debugging aids?